### PR TITLE
ServerTest verifies the number of NioSocketChannel's

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

**Comment by sullis** [comment]
wdyt? @argha-c

File: zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java [hunk 1]
```diff
@@ -68,9 +71,20 @@ public void beforeTest() {
     public void getListeningSockets() throws Exception {
         ServerStatusManager ssm = mock(ServerStatusManager.class);
         Map<NamedSocketAddress, ChannelInitializer<?>> initializers = new HashMap<>();
+        final List<NioSocketChannel> nioChannels = Collections.synchronizedList(new ArrayList<NioSocketChannel>());
```
Comment (argha-c) [inline zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java:74]: Why is this specific to NIO channels?

Comment (sullis) [reply zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java:74]: > Why is this specific to NIO channels?

ServerTest uses **force_nio**  true

```
config.setProperty("zuul.server.netty.socket.force_nio", "true");
```

so the channels should be NioSocketChannel objects

Comment (argha-c) [reply zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java:74]: Ah, I forgot we updated that in the last change.

**Review by argha-c — APPROVED** [review]

**Pull request merged**